src/runtime/manifest-parser.pegjs

```diff
@@ -1668,15 +1668,12 @@ ExpressionWithQualifier
   }
 
 PaxelExpression
-  = expr:(NewExpression / ExpressionWithQualifier) {
+  = expr:(NewExpression / ExpressionWithQualifier / RefinementExpression) {
```

Comment:
as mentioned in the other PR, this is less precise than needed, you could modify
```
  / fn: (FunctionCall / fieldName) nested:(('.' / '?.') fieldName)*
```

in primary expression into it's own rule

```
scopeLookup = fn: (FunctionCall / fieldName) nested:(('.' / '?.') fieldName)*
```

and then add it to the PaxelExpression rule. But if you want to accept stuff like '= 1 + 2', you might want to add an assert on the Evaluator Particle that the top level expression must be either a QualifiedExpression or a FieldExpression to catch it at runtime.

Reply:
As discussed f2f: "1 + 2" is a valid paxel expression. We can represent it in paxel nodes, we parse it ok with Kotlin PaxelParser etc.

Where I think output: `writes Foo {bar: Text} = 1 + 2` should be caught is in the type checking, as obviously Entity Foo is not a number.